src/responsive-grid.scss

```diff
@@ -0,0 +1,201 @@
+$spacer: 4px !default;
+
+$grid-gutter: $spacer * 6 !default;
```

Comment:
Out of curiosity: Why `6`?

Reply:
Just for a nicer width gap for our container, it can be modified without a problem

Reply:
maybe we should create a variable for it